### ISSUE TITLE
Null-Pointer Exception on RN 0.33?

### ISSUE THREAD

**Issue by orhan** [orhan/react-native-android-blurryoverlay#2]
See https://github.com/orhan/react-native-android-blurryoverlay/pull/1#issuecomment-244776063

**Comment by priezz** [comment]
I have managed this to work without Activity, works fine with RN 0.33. Will make a PR later this week.

**Comment by rammuk** [comment]
@priezz How did you manage to work without Activity 
I am getting the same error:
Attempt to invoke virtual method 'android.view.View android.app.Activity.findViewById(int)' on a null

Thanks

**Comment by priezz** [comment]
Yes, I did. However, as I remember I have changed the logic of the module (applied it to parent Image only). Unfortunately I decided not to use blur at all because of the very slow processing - there is a noticeable delay (half a second or something) before the effect is applied and that looks really ugly from the user's side.

**Comment by rammuk** [comment]
@priezz Thanks Ill give it a try
